### PR TITLE
fitContent props for textarea component

### PR DESCRIPTION
**Pull request by pvilalta**
Hello all,

This PR lead to the TextArea component proposing a new props that allow us to make the content fit the textbox (removing scrollbar)

Screenshot 2022-01-07 at 10 07 44
 
Paul

### CODE REVIEW

File: src/web/views/Components/Textarea/Textarea.vue [hunk 1]
```diff
@@ -164,6 +175,13 @@ export default {
         return false;
       }
     },
+    getFitContent() {
```
[the remark this answers is not in the document]
Comment (pvilalta) [reply src/web/views/Components/Textarea/Textarea.vue:178]: indeed, we can get rid of the getFitContent() method and replace it call line 22 by "fitContent"
thank you for your input

**Comment by pvilalta** [comment]
Hello All,

Some modifications have been made

* code enhancement for the fitContent props
* add of a new props -> resizable

Paul

**Review by Kurtil — APPROVED** [review]

**Pull request merged**